ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_ospf_max_lsa.schema.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  underlay_ospf_max_lsa:
+    documentation_options:
+      filename: Fabric Variables
+      table: OSPF Settings
+    description: Underlay OSFP Required when < underlay_routing_protocol > == OSPF variants
```

Comment:
When we have a default value the key is not required. Also typo O*SFP*.
Same elsewhere.

Reply:
good catch, removed this incorrect description